packages/api/src/services/stats/lsp.ts

```diff
@@ -0,0 +1,165 @@
+import assert from "assert";
```

Comment:
this calculates the stats (tvl) for LSP.  This is almost verbatim from the EMP stats service with unnecessary calls removed. Theres so much code duplication and this is a good candidate to refactor to try and combine the logic for both contract types. Requires some more time and thought to do this, and probably would need to break out each stat type into its own file (tvl, tvm)

Reply:
ye, for sure would be ideal to join these somehow in the future